src/pattern-library/components/patterns/ButtonPatterns.js

```diff
@@ -81,6 +81,48 @@ export default function ButtonPatterns() {
           </PatternExample>
         </PatternExamples>
 
+        <h3>Dark variant</h3>
+        <p>
+          This variant is for use on darker (light grey vs. white) backgrounds.
+          Note that this button, unlike other <code>IconButton</code>s, has a
+          background color. This is to allow for a use case in which the dark{' '}
+          <code>IconButton</code> is initially fixed on a grey background but
+          floats on top of content when scrolled.
```

Comment:
That's nice that you made it work straight for the notebook, thanks! Do you think could be a problem in other context?

Reply:
I appreciate you asking this! And I'll be honest to say: I don't know! There is a little bit of experimentation going on here. We might find that we want to harmonize this by adding an explicit background color for `LabeledButton`'s `dark` variant.